docs/tryCatch.md

```diff
@@ -0,0 +1,26 @@
+# _.tryCatch(catchFunction, trySupplier)
+
+Invokes the `trySupplier` in a `try..catch` block and invoked the `catchFunction` with the caught `error`.
+
+## Arguments
+
+* `catchFunction (function)`: Function to map the caught `error`.
+* `trySupplier (function)`: Nullary function that may or may not throw an `Error`.
+
+## Returns
+
+* `(*)`: The value provided by the `trySupplier` or the return value from the `catchFunction`.
+
+## Examples
+
+### Try and catch
+
+```javascript
+const tryWithoutError = _.tryCatch(F.constant(null)); // Throw away error
+
+tryWithoutError(() => fs.readFileSync(foundFile, { encoding: "utf8" }));
+// => data
+
+tryWithoutError(() => fs.readFileSync(missingFile, { encoding: "utf8" }));
+// => null
+```
```

Comment:
Do you not need an example of invoking it with a function to process the error?

Reply:
Updated.